quickstart_dense_test.go

```diff
@@ -108,15 +108,19 @@ func readDenseArray() {
 
 	// Print out the results.
 	fmt.Println(data)
+
+	return data
 }
 
 // ExampleDenseArray shows and example creation, writing and reading of a dense
 // array
-func Example_denseArray() {
+func TestDenseArray(t *testing.T) {
```

Comment:
The function name should start with `Example.`. instead of `Test..` so it shows up as an [example in the documentation](https://godoc.org/github.com/TileDB-Inc/TileDB-Go#ex-package--DenseArray). Example are suppose to print their output and that is how it is check.

See https://blog.golang.org/examples